src/pages/en/installation-new.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Installation
+description: How to install Astro with NPM, PNPM, or Yarn.
+---
+## Prerequisites
+
+- **Node.js** - `14.15.0`, `v16.0.0`, or higher.
+- **Text editor** - We recommend [VS Code](https://code.visualstudio.com/) with our [Official Astro extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode).
+- **Terminal** - Astro is mainly accessed via the terminal’s command-line.
+
+## Install Astro
+
+### Automatic Setup
+
+1. Run one of the following commands in your terminal to start the `create-astro` install wizard.
+
+```shell
+# npm
+npm init astro
+
+# yarn
+yarn create astro
+
+# pnpm
+pnpm create astro
+```
+
+The installation wizard will allow you to
+- Choose from a set of [starter templates](https://github.com/withastro/astro/tree/main/examples) (e.g. basic starter, blog, portfolio, documentation)
+- Select which [additional frameworks](/en/core-concepts/component-hydration) (React, Svelte, Vue, Solid, Preact), if any, you would like to include in your project
+
+2. Install your project's dependencies using npm or your package manager of choice.
+
+```bash
+# npm
+npm install
+
+# yarn
+yarn
+
+#pnmp
+pnmp install
+
+```
+
+<details>
+<summary style="font-weight: bold; font-size: 1.5em;">Prefer to install manually?</summary>
```

Comment:
I'm not sure if I love hidden docs like this. I think we can just treat this as two different sections for now, and make sure that it's clear that you can jump over this when you get to the end of "Automatic Install".

For sidebar table-of-contents consistency, this could be:

```diff
 ## Install Astro
 ### Automatic Setup
-### Manual Install
+### Manual Setup
```

(or something similar)

Reply:
Or, if you really wanted to get ahead of things, you could move this "Manual Install" out to a new "Tutorial" page that you could then link to from here. But I bet that's a bit out of scope for this PR, so just a thought.

I think the reason this popped into my head is that some of this content feels well outside the scope of "installation" and more in the realm of "here's a step-by-step on how to get started"

Reply:
The accordion is considered good doc practice to hide clutter!  ;) 

Of the two options, my preference is to move manual setup out to its own guide. Since we're not really "losing" these instructions (this PR only adds a -new page, doesn't remove the old one, and doesn't replace the exisiting on the live site), I'm just going to remove manual setup instructions from here.

 I'll make another PR that includes a new Manual Setup page, and at some point, will swap the existing Installation page out for this new one.